test/bootstrap.php

```diff
@@ -21,12 +21,18 @@
     unset($phpUnitVersion);
 }
 
+
 /**
  * Setup autoloading
  */
 require __DIR__ . '/../vendor/autoload.php';
 
 /**
+ * Setting the Zend\StdLib/ErrorHandler as default ErrorHandler
+ */
+Zend\Ldap\ErrorHandler::setErrorHandler(new ZendTest\Ldap\ErrorHandler());
```

Comment:
As the default ErrorHandler does nothing at all but the tests seem to depend on error-suppression I've added that one to get the tests running under the same conditions as before. I was thinking of adding that line to src/Ldap.php (at the very end) so that that's set as default error handler and users can overwrite that with their own implementation. On the other hand that would mean that the dependency to ```Zend\StdLib\ErrorHandler``` would not be eliminated.